### PR TITLE
nginx.md: Add note about potential security isues

### PR DESCRIPTION
**Pull request by perlun**
I thought about this while setting this up, and then found this guide (I was setting it up without the guide first.)

The potential security implications are important, so I think we should mention them here on this web page. (We could even go further by outright _warning_ people about this, but perhaps letting people know about it so they can make an informed decision is a better way to go. This can be perfectly fine for certain intranet scenarios.)

### CODE REVIEW

**Comment by paigehargrave** [comment]
@ahh-docker - Needs wording updates.

**Comment by perlun** [comment]
@paigehargrave Sure, feel free to suggest improvements.

**Comment by ahh-docker** [comment]
@thaJeztah PTAL.

**Comment by bermudezmt** [comment]
@thaJeztah can you please have a look? :)

File: registry/recipes/nginx.md [hunk 1]
```diff
@@ -38,11 +38,10 @@ you want through the secondary authentication mechanism implemented inside your
...
 proxy itself.
 
-> Another important thing to note is that by binding your registry to
-> `localhost:5000` without authentication, you open up a potential loophole in
-> your Docker Registry security - anyone who can log on to the server where your
-> Docker Registry is running can push images to your registry, without
-> authentication. This could have potentially devastating effects.
+> ***NOTE:*** Docker does not recommend binding your registry to `localhost:5000` without
+> authentication. This creates a potential loophole in your Docker Registry security.
+> As a result, anyone with access to your Docker Registry can push images without
```
Comment (kolyshkin) [inline registry/recipes/nginx.md]: s/anyone with access to your Docker Registry/anyone who can log on to the server where your Docker Registry is running/

**Comment by kolyshkin** [comment]
> anyone with access to your Docker Registry can push images...

This is true by definition (this is what Registry is for -- pushing (and pulling) images). So the phrase does not make much sense. It's better to say the way it was said before, i.e. "anyone who can log on to the server where your Docker Registry is running can push images..."

Otherwise LGTM

File: registry/recipes/nginx.md (hunk 1 above)
Comment (ahh-docker) [reply registry/recipes/nginx.md]: resolved

**Comment by ahh-docker** [comment]
@L-Hudson PTAL.

**Pull request merged**